src/js/common/polyfills.js

```diff
@@ -0,0 +1,26 @@
+// Common Javascript environment setup to be used by all entrypoints.
+//
+// Keep this short. It should mostly be polyfills, global site style, and any
+// truly pervasive libraries. Be careful if you are tempted to put jquery or
+// other such libraries in here. Most of the site does not use these legacy
+// frameworks and it belongs in a lower-level module.
+
+require('babel-polyfill');
```

Comment:
By what I'm reading [here](http://2ality.com/2017/02/babel-preset-env.html#usebuiltins-boolean-default-false), it looks like that's true for both `babel-polyfill` and `core-js`.

The only difference I see is this:
> `babel-polyfill` polyfills `core-js` and the regenerator runtime (to emulate generators on ES5).

Is there any other reason to choose one over the other?